### ISSUE TITLE
integration::worker::workers is flaky

### ISSUE THREAD

**Issue by ry** [denoland/deno#12075]
```
2021-09-14T21:57:04.0448811Z test worker location ...
2021-09-14T21:57:04.0449620Z OUTPUT
2021-09-14T21:57:04.0453133Z ##[error]thread 'integration::worker::workers' panicked at 'bad exit code, expected: 0, actual: 1', test_util\src\lib.rs:1493:9
2021-09-14T21:57:04.0455255Z stack backtrace:
2021-09-14T21:57:04.0456292Z    0:     0x7ff6721ceaa4 - std::backtrace_rs::backtrace::dbghelp::trace
2021-09-14T21:57:04.0457934Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\..\..\backtrace\src\backtrace\dbghelp.rs:98
2021-09-14T21:57:04.0460607Z    1:     0x7ff6721ceaa4 - std::backtrace_rs::backtrace::trace_unsynchronized
2021-09-14T21:57:04.0462265Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\..\..\backtrace\src\backtrace\mod.rs:66
2021-09-14T21:57:04.0463736Z    2:     0x7ff6721ceaa4 - std::sys_common::backtrace::_print_fmt
2021-09-14T21:57:04.0465217Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys_common\backtrace.rs:67
2021-09-14T21:57:04.0466649Z    3:     0x7ff6721ceaa4 - std::sys_common::backtrace::_print::impl$0::fmt
2021-09-14T21:57:04.0468204Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys_common\backtrace.rs:46
2021-09-14T21:57:04.0469509Z    4:     0x7ff67205cc1a - core::fmt::write
2021-09-14T21:57:04.0470840Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\core\src\fmt\mod.rs:1115
2021-09-14T21:57:04.0472470Z    5:     0x7ff6721ce338 - std::io::Write::write_fmt<alloc::vec::Vec<u8,alloc::alloc::Global> >
2021-09-14T21:57:04.0474052Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\io\mod.rs:1665
2021-09-14T21:57:04.0475344Z    6:     0x7ff6721cdc02 - std::sys_common::backtrace::_print
2021-09-14T21:57:04.0477331Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys_common\backtrace.rs:49
2021-09-14T21:57:04.0478714Z    7:     0x7ff6721cdc02 - std::sys_common::backtrace::print
2021-09-14T21:57:04.0482174Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys_common\backtrace.rs:36
2021-09-14T21:57:04.0483601Z    8:     0x7ff6721cdc02 - std::panicking::default_hook::closure$1
2021-09-14T21:57:04.0485042Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:208
2021-09-14T21:57:04.0486384Z    9:     0x7ff6721cce80 - std::panicking::default_hook
2021-09-14T21:57:04.0487772Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:222
2021-09-14T21:57:04.0489223Z   10:     0x7ff6721cce80 - std::panicking::rust_panic_with_hook
2021-09-14T21:57:04.0490645Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:622
2021-09-14T21:57:04.0492038Z   11:     0x7ff6721d5395 - std::panicking::begin_panic_handler::closure$0
2021-09-14T21:57:04.0493501Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:519
2021-09-14T21:57:04.0495250Z   12:     0x7ff6721d52ef - std::sys_common::backtrace::__rust_end_short_backtrace<std::panicking::begin_panic_handler::closure$0,never$>
2021-09-14T21:57:04.0497005Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys_common\backtrace.rs:141
2021-09-14T21:57:04.0498406Z   13:     0x7ff6721d528f - std::panicking::begin_panic_handler
2021-09-14T21:57:04.0499876Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:515
2021-09-14T21:57:04.0501179Z   14:     0x7ff672444d30 - std::panicking::begin_panic_fmt
2021-09-14T21:57:04.0502588Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\panicking.rs:457
2021-09-14T21:57:04.0504340Z   15:     0x7ff6722353d7 - core::fmt::impl$0::write_fmt<std::io::Write::write_fmt::Adaptor<enum$<test::console::OutputLocation<std::io::stdio::Stdout> > > >
2021-09-14T21:57:04.0505523Z   16:     0x7ff671f5c4b3 - main
2021-09-14T21:57:04.0506446Z   17:     0x7ff67220eaa6 - core::ops::function::FnOnce::call_once
2021-09-14T21:57:04.0507889Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\core\src\ops\function.rs:227
2021-09-14T21:57:04.0509334Z   18:     0x7ff67220eaa6 - test::__rust_begin_short_backtrace<void (*)()>
2021-09-14T21:57:04.0510751Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\test\src\lib.rs:578
2021-09-14T21:57:04.0512018Z   19:     0x7ff67220ed68 - alloc::boxed::impl$44::call_once
2021-09-14T21:57:04.0513373Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\alloc\src\boxed.rs:1572
2021-09-14T21:57:04.0514651Z   20:     0x7ff67220ed68 - std::panic::impl$30::call_once
2021-09-14T21:57:04.0515986Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panic.rs:347
2021-09-14T21:57:04.0517245Z   21:     0x7ff67220ed68 - std::panicking::try::do_call
2021-09-14T21:57:04.0518609Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panicking.rs:401
2021-09-14T21:57:04.0519905Z   22:     0x7ff67220ed68 - std::panicking::try
2021-09-14T21:57:04.0521427Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panicking.rs:365
2021-09-14T21:57:04.0522695Z   23:     0x7ff67220ed68 - std::panic::catch_unwind
2021-09-14T21:57:04.0524101Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panic.rs:434
2021-09-14T21:57:04.0525379Z   24:     0x7ff67220ed68 - test::run_test_in_process
2021-09-14T21:57:04.0526696Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\test\src\lib.rs:601
2021-09-14T21:57:04.0528010Z   25:     0x7ff67220ed68 - test::run_test::run_test_inner::closure$0
2021-09-14T21:57:04.0530270Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\test\src\lib.rs:493
2021-09-14T21:57:04.0531594Z   26:     0x7ff672210d54 - test::run_test::run_test_inner::closure$1
2021-09-14T21:57:04.0532980Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\test\src\lib.rs:520
2021-09-14T21:57:04.0534620Z   27:     0x7ff672210d54 - std::sys_common::backtrace::__rust_begin_short_backtrace<test::run_test::run_test_inner::closure$1,tuple$<> >
2021-09-14T21:57:04.0536352Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\sys_common\backtrace.rs:125
2021-09-14T21:57:04.0537834Z   28:     0x7ff672210aa5 - std::thread::impl$0::spawn_unchecked::closure$0::closure$0
2021-09-14T21:57:04.0539387Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\thread\mod.rs:476
2021-09-14T21:57:04.0540640Z   29:     0x7ff672210aa5 - std::panic::impl$30::call_once
2021-09-14T21:57:04.0541989Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panic.rs:347
2021-09-14T21:57:04.0543223Z   30:     0x7ff672210aa5 - std::panicking::try::do_call
2021-09-14T21:57:04.0544590Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panicking.rs:401
2021-09-14T21:57:04.0545844Z   31:     0x7ff672210aa5 - std::panicking::try
2021-09-14T21:57:04.0547173Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panicking.rs:365
2021-09-14T21:57:04.0548437Z   32:     0x7ff672210aa5 - std::panic::catch_unwind
2021-09-14T21:57:04.0549798Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\panic.rs:434
2021-09-14T21:57:04.0551167Z   33:     0x7ff672210aa5 - std::thread::impl$0::spawn_unchecked::closure$0
2021-09-14T21:57:04.0552591Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\std\src\thread\mod.rs:475
2021-09-14T21:57:04.0554232Z   34:     0x7ff672210aa5 - core::ops::function::FnOnce::call_once<std::thread::impl$0::spawn_unchecked::closure$0,tuple$<> >
2021-09-14T21:57:04.0555863Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\core\src\ops\function.rs:227
2021-09-14T21:57:04.0557188Z   35:     0x7ff6721ef4cb - alloc::boxed::impl$44::call_once
2021-09-14T21:57:04.0558541Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\alloc\src\boxed.rs:1572
2021-09-14T21:57:04.0559877Z   36:     0x7ff6721ef4cb - alloc::boxed::impl$44::call_once
2021-09-14T21:57:04.0561254Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\library\alloc\src\boxed.rs:1572
2021-09-14T21:57:04.0562627Z   37:     0x7ff6721ef4cb - std::sys::windows::thread::impl$0::new::thread_start
2021-09-14T21:57:04.0564137Z                                at /rustc/c8dfcfe046a7680554bf4eb612bad840e7631c4b\/library\std\src\sys\windows\thread.rs:57
2021-09-14T21:57:04.0565484Z   38:     0x7ffe54b27974 - BaseThreadInitThunk
2021-09-14T21:57:04.0566537Z   39:     0x7ffe5793a2f1 - RtlUserThreadStart
2021-09-14T21:57:04.0567092Z 
2021-09-14T21:57:04.0567447Z 
2021-09-14T21:57:04.0568194Z failures:
2021-09-14T21:57:04.0568973Z     integration::worker::workers
2021-09-14T21:57:04.0569520Z 
2021-09-14T21:57:04.0570674Z test result: FAILED. 515 passed; 1 failed; 9 ignored; 0 measured; 0 filtered out; finished in 433.56s
2021-09-14T21:57:04.0571402Z 
```
https://github.com/denoland/deno/runs/3603807319

**Comment by bartlomieju** [comment]
Ref https://github.com/denoland/deno/issues/11350

**Comment by andreubotella** [comment]
The panic is the same as in #11342, and it seems to be related to some of the nested worker tests (see https://github.com/denoland/deno/issues/11342#issuecomment-918327693). This looks like a race condition between the time that the middle worker's `JsRuntime` gets dropped and the time that the nested worker's event loop finishes running.

**Comment by ry** [comment]
I hit this again https://github.com/denoland/deno/pull/12175/checks?check_run_id=3676146750
```
2021-09-22T14:57:58.9171330Z test integration::worker::workers ... FAILED
2021-09-22T14:57:58.9172430Z 
2021-09-22T14:57:58.9173100Z failures:
2021-09-22T14:57:58.9173540Z 
2021-09-22T14:57:58.9176380Z ---- integration::worker::workers stdout ----
2021-09-22T14:57:58.9177740Z deno_exe path /Users/runner/work/deno/deno/target/release/deno
2021-09-22T14:57:58.9180430Z deno_exe args test --reload --location http://127.0.0.1:4545/ --allow-net --allow-read --unstable workers/test.ts
2021-09-22T14:57:58.9182540Z deno_exe testdata path "/Users/runner/work/deno/deno/cli/tests/testdata"
2021-09-22T14:57:58.9183590Z OUTPUT
2021-09-22T14:57:58.9186040Z [0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/test.ts
2021-09-22T14:57:58.9188690Z running 30 tests from file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/test.ts
2021-09-22T14:57:58.9191920Z [0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/test_worker.ts
2021-09-22T14:57:58.9194840Z test worker terminate ... [0m[32mok[0m [0m[38;5;8m(1168ms)[0m
2021-09-22T14:57:58.9218810Z [0m[32mCheck[0m data:application/typescript;base64,aWYgKHNlbGYubmFtZSAhPT0gInRzV29ya2VyIikgewogIHRocm93IEVycm9yKGBJbnZhbGlkIHdvcmtlciBuYW1lOiAke3NlbGYubmFtZX0sIGV4cGVjdGVkIHRzV29ya2VyYCk7Cn0KCm9ubWVzc2FnZSA9IGZ1bmN0aW9uIChlKTogdm9pZCB7CiAgcG9zdE1lc3NhZ2UoZS5kYXRhKTsKICBjbG9zZSgpOwp9Owo=
2021-09-22T14:57:58.9241770Z test worker from data url ... [0m[32mok[0m [0m[38;5;8m(834ms)[0m
2021-09-22T14:57:58.9244320Z test worker nested ... [0m[32mok[0m [0m[38;5;8m(44ms)[0m
2021-09-22T14:57:58.9246390Z [0m[1m[31merror[0m: Uncaught (in worker "") Error: Thrown error
2021-09-22T14:57:58.9247440Z throw Error("Thrown error");
2021-09-22T14:57:58.9249480Z [0m[31m      ^[0m
2021-09-22T14:57:58.9251830Z     at [0m[36mfile:///Users/runner/work/deno/deno/cli/tests/testdata/workers/throwing_worker.js[0m:[0m[33m2[0m:[0m[33m7[0m
2021-09-22T14:57:58.9254430Z test worker throws when executing ... [0m[32mok[0m [0m[38;5;8m(34ms)[0m
2021-09-22T14:57:58.9256970Z [0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/worker_globals.ts
2021-09-22T14:57:58.9259250Z test worker globals ... [0m[32mok[0m [0m[38;5;8m(824ms)[0m
2021-09-22T14:57:58.9261300Z test worker fetch API ... [0m[32mok[0m [0m[38;5;8m(67ms)[0m
2021-09-22T14:57:58.9263500Z test worker terminate busy loop ... [0m[32mok[0m [0m[38;5;8m(217ms)[0m
2021-09-22T14:57:58.9265680Z test worker race condition ... [0m[32mok[0m [0m[38;5;8m(309ms)[0m
2021-09-22T14:57:58.9267850Z test worker is event listener ... [0m[32mok[0m [0m[38;5;8m(34ms)[0m
2021-09-22T14:57:58.9270950Z test worker scope is event listener ... [0m[32mok[0m [0m[38;5;8m(45ms)[0m
2021-09-22T14:57:58.9273550Z [0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/deno_worker.ts
2021-09-22T14:57:58.9276840Z test worker with Deno namespace ... [0m[32mok[0m [0m[38;5;8m(1370ms)[0m
2021-09-22T14:57:58.9278130Z test worker with crypto in scope ... [0m[32mok[0m [0m[38;5;8m(35ms)[0m
2021-09-22T14:57:58.9279370Z test Worker event handler order ... [0m[32mok[0m [0m[38;5;8m(28ms)[0m
2021-09-22T14:57:58.9280590Z test Worker immediate close ... [0m[32mok[0m [0m[38;5;8m(1022ms)[0m
2021-09-22T14:57:58.9282060Z [0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/post_undefined.ts
2021-09-22T14:57:58.9282860Z test Worker post undefined ...
2021-09-22T14:57:58.9283750Z post from parent
2021-09-22T14:57:58.9284090Z 
2021-09-22T14:57:58.9284990Z received in worker [1mnull[22m
2021-09-22T14:57:58.9285500Z posted from worker
2021-09-22T14:57:58.9286380Z main <- worker: [1mnull[22m
2021-09-22T14:57:58.9286680Z 
2021-09-22T14:57:58.9287450Z  [0m[32mok[0m [0m[38;5;8m(790ms)[0m
2021-09-22T14:57:58.9288580Z test Worker inherits permissions ... [0m[32mok[0m [0m[38;5;8m(37ms)[0m
2021-09-22T14:57:58.9289900Z test Worker limit children permissions ... [0m[32mok[0m [0m[38;5;8m(27ms)[0m
2021-09-22T14:57:58.9292200Z test Worker limit children permissions granularly ... [0m[32mok[0m [0m[38;5;8m(51ms)[0m
2021-09-22T14:57:58.9294340Z test Nested worker limit children permissions ... [0m[32mok[0m [0m[38;5;8m(47ms)[0m
2021-09-22T14:57:58.9320890Z ##[error]thread 'worker-0' panicked at 'coding error: either js is polling or the worker is terminated', runtime/web_worker.rs:547:9
2021-09-22T14:57:58.9334530Z stack backtrace:
2021-09-22T14:57:58.9336360Z test Nested worker limit children permissions granularly ... [0m[32mok[0m [0m[38;5;8m(52ms)[0m
2021-09-22T14:57:58.9338200Z test Worker initialization throws on worker permissions greater than parent thread permissions ... [0m[32mok[0m [0m[38;5;8m(8ms)[0m
2021-09-22T14:57:58.9339770Z test Worker with disabled permissions ... [0m[32mok[0m [0m[38;5;8m(36ms)[0m
2021-09-22T14:57:58.9341670Z    0:        0x102d41c73 - <std::sys_common[0m[32mCheck[0m file:///Users/runner/work/deno/deno/cli/tests/testdata/workers/worker_location.ts
2021-09-22T14:57:58.9342910Z ::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h8d168763f08a78b9
2021-09-22T14:57:58.9344420Z    1:        0x1025fe00c - core::fmt::write::h0fa46d4891ed63f1
2021-09-22T14:57:58.9345840Z    2:        0x102d3fb6a - std::io::Write::write_fmt::hacbf9e10e17149ce
2021-09-22T14:57:58.9347380Z    3:        0x102d40daf - std::panicking::default_hook::{{closure}}::hee7cf38dedfa76fc
2021-09-22T14:57:58.9348910Z    4:        0x102d3ff7d - std::panicking::default_hook::hd12252412832a45f
2021-09-22T14:57:58.9350750Z    5:        0x1025bbe3b - deno::setup_exit_process_panic_hook::{{closure}}::hfc14741de1f7af1c
2021-09-22T14:57:58.9353090Z    6:        0x102d3fa19 - std::panicking::rust_panic_with_hook::h521fd09e07057893
2021-09-22T14:57:58.9355440Z    7:        0x1027ea61d - std::panicking::begin_panic::{{closure}}::hbfb9e23776e6eca5
2021-09-22T14:57:58.9357920Z    8:        0x1027ea5e8 - std::sys_common::backtrace::__rust_end_short_backtrace::h1db2097df5b00baf
2021-09-22T14:57:58.9360470Z    9:        0x103ecba1d - std::panicking::begin_panic::he9b3445218827e53
2021-09-22T14:57:58.9362210Z   10:        0x10291a60a - deno_runtime::web_worker::WebWorker::poll_event_loop::hbf9e1a41f08fe63e
2021-09-22T14:57:58.9363880Z   11:        0x102918a68 - deno_runtime::web_worker::run_web_worker::{{closure}}::h4bd7d6e9028a3f53
2021-09-22T14:57:58.9365520Z   12:        0x1029151a4 - std::sys_common::backtrace::__rust_begin_short_backtrace::h91f78691d5810739
2021-09-22T14:57:58.9367170Z   13:        0x102914166 - core::ops::function::FnOnce::call_once{{vtable.shim}}::hd4dab586b384ef20
2021-09-22T14:57:58.9368790Z   14:        0x102d741aa - std::sys::unix::thread::Thread::new::thread_start::h0a9760ff4c2a0824
2021-09-22T14:57:58.9369850Z   15:     0x7fff71183109 - __pthread_start
2021-09-22T14:57:58.9370420Z test worker location ...
2021-09-22T14:57:58.9370830Z OUTPUT
2021-09-22T14:57:58.9379460Z ##[error]thread 'integration::worker::workers' panicked at 'bad exit code, expected: 0, actual: 1', test_util/src/lib.rs:1491:9
2021-09-22T14:57:58.9381050Z stack backtrace:
2021-09-22T14:57:58.9382840Z    0:        0x1025ee4b3 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h8d168763f08a78b9
2021-09-22T14:57:58.9384410Z    1:        0x10249a38c - core::fmt::write::h0fa46d4891ed63f1
2021-09-22T14:57:58.9385810Z    2:        0x1025edcea - std::io::Write::write_fmt::hb1b5ffef1642a646
2021-09-22T14:57:58.9394860Z    3:        0x1025ed5ef - std::panicking::default_hook::{{closure}}::hee7cf38dedfa76fc
2021-09-22T14:57:58.9396570Z    4:        0x1025ec5e6 - std::panicking::rust_panic_with_hook::h521fd09e07057893
2021-09-22T14:57:58.9398550Z    5:        0x10260a94a - std::panicking::begin_panic_handler::{{closure}}::h3588a5a9cca9e8e5
2021-09-22T14:57:58.9401010Z    6:        0x10260a8b8 - std::sys_common::backtrace::__rust_end_short_backtrace::h86522c151a195ff9
2021-09-22T14:57:58.9402670Z    7:        0x10260a860 - _rust_begin_unwind
2021-09-22T14:57:58.9405180Z    8:        0x10284710f - std::panicking::begin_panic_fmt::hf4ebaa1b90291b0b
2021-09-22T14:57:58.9407680Z    9:        0x10265d55a - test_util::CheckOutputIntegrationTest::run::h003f8430936e4046
2021-09-22T14:57:58.9410120Z   10:        0x1023ca8aa - core::ops::function::FnOnce::call_once::h34952827709dcbb9
2021-09-22T14:57:58.9412360Z   11:        0x10263906a - test::__rust_begin_short_backtrace::h918af6527b4f3a5f
2021-09-22T14:57:58.9414700Z   12:        0x10263930b - test::run_test::run_test_inner::{{closure}}::h76a4cff99fa950a4
2021-09-22T14:57:58.9417180Z   13:        0x10263ad8d - std::sys_common::backtrace::__rust_begin_short_backtrace::h7649c4fbea4fb790
2021-09-22T14:57:58.9419730Z   14:        0x10263abd2 - core::ops::function::FnOnce::call_once{{vtable.shim}}::h84dcca60f7717f46
2021-09-22T14:57:58.9421920Z   15:        0x10262076a - std::sys::unix::thread::Thread::new::thread_start::h0a9760ff4c2a0824
2021-09-22T14:57:58.9422970Z   16:     0x7fff71183109 - __pthread_start
2021-09-22T14:57:58.9423310Z 
2021-09-22T14:57:58.9423520Z 
2021-09-22T14:57:58.9423900Z failures:
2021-09-22T14:57:58.9424400Z     integration::worker::workers
```

**Comment by andreubotella** [comment]
Should be fixed with #12156
